Header header
uint8 data1
uint8 fan_speed
uint8 data6
uint8 data7
uint8 data8

uint8 data9
uint8 defrost_fan0
uint8 defrost_fan1
uint8 data12
uint8 data13
uint8 defrost_1
uint8 defrost_2
uint8 data16

bool auto
bool system_on
bool unit_on
bool dual
bool max_cool
bool max_defrost
bool recirculation
bool head_fan
bool feet_fan
bool front_defrost
bool rear_defrost

string driver_temp
string passenger_temp
